gapir/cc/BUILD.bazel

```diff
@@ -45,10 +45,7 @@ mm_library(
     name = "darwin_renderer",
     srcs = glob(["osx/*.mm"]),
     defines = cc_defines(),
-    deps = [
-        ":gles_lib",
-        ":vulkan_lib",
-    ],
+    deps = ["api_lib"],
```

Comment:
I think this needs the : otherwise you get wonderfully fragile behavior of guessing whether it is a file or label...

Reply:
Done. Thanks.